Assets/Scripts/DogAI/DogAI.cs

```diff
@@ -0,0 +1,116 @@
+﻿/*
+ * Author(s): Timothy Ng
+ * Description: Brains behind the dogs actions on the world
+ * Usage: [no notes]
+ */
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using k = PPGlobal;
+
+public class DogAI : MonoBehaviour {
+
+    enum State
+    {
+        Idle,
+        Wandering,
+        Eating,
+        Pooping
+    }
+
+    State currentState = State.Wandering;
+    Vector2 wanderCenter;
+    float wanderRadius = 500;
+
+    Vector2 target;
+    int frameCounter = 0;
+
+    int tapCount = 0;
+
+	// Use this for initialization
+	void Start () {
+        wanderCenter = GetComponent<RectTransform>().anchoredPosition;
+        target = GetComponent<RectTransform>().anchoredPosition;
+    }
+	
+	void Update () {
+
+        currentState = DecideState();
+
+        switch (currentState)
+        {
+            case State.Idle:
+                break;
+            case State.Wandering:
+                Wander();
+                break;
+            case State.Eating:
+                break;
+            case State.Pooping:
+                break;
+            default:
+                break;
+        }
+	}
+
+    State DecideState()
+    {
+        if(frameCounter < 120)
+        {
+            frameCounter++;
+            return currentState;
+        }
+        else
+        {
+            frameCounter = 0;
+            switch(Random.Range(0, 2))
```

Comment:
This part of the code was to demo what the AI can do for the build, Im going to scrap everything in that block once design decides what they want dogs to do and when